cluster/ubuntu/util.sh

```diff
@@ -508,6 +505,12 @@ function provision-masterandnode() {
       cp ~/kube/minion/* /opt/bin/
 
       service etcd start
+      service kube-apiserver start
```

Comment:
OK, great idea. But it does not work on ubuntu 15.10. I'd like to write some systemd unit files.
I will close this pull requests and create a new pull request on finish.

Thanks